app/controllers/api_controller/providers.rb

```diff
@@ -5,7 +5,7 @@ module Providers
     CREDENTIALS_ATTR  = "credentials"
     AUTH_TYPE_ATTR    = "auth_type"
     DEFAULT_AUTH_TYPE = "default"
-    ENDPOINT_ATTRS    = %w(hostname ipaddress port)
+    ENDPOINT_ATTRS    = %w(hostname ipaddress port security_protocol).freeze
```

Comment:
I see other attributes being delegated to Endpoint in the newer providers (app/models/providers.rb), like verify_ssl, url, pah, etc. I wonder if we shouldn't get this list from the Endpoint model, i.e. Endpoint.columns_hash.keys or something like Endpoint.supported_attributes